bip-0159.mediawiki

```diff
@@ -34,6 +34,8 @@ This BIP proposes a new service bit
 | NODE_NETWORK_LIMITED || bit 10 (0x400) || If signaled, the peer <I>MUST</I> be capable of serving at least the last 288 blocks (~2 days).
 |}
 
+Pruned/limited peers <I>MUST NOT</I> set a service bit that signals serving the complete block chain (e.g., <code>NODE_NETWORK</code>.
```

Comment:
Should we add an info that "normal" full nodes (`NODE_NETWORK`) can also signal `NODE_NETWORK_LIMITED` as it is inclusive (which would reflect the current behavior on the network)?